list.go

```diff
@@ -65,3 +65,9 @@ func (l *List) Actions() (actions []Action, err error) {
 	}
 	return
 }
+
+// AddCard is just a wrapper for Client.NewCard
+func (l *List) AddCard(card Card) (*Card, error) {
```

Comment:
This is the API that I really like. I don't think we need `client.NewCard()`, since it always has to be created on a specific and existing list.